flink-runtime/src/main/java/org/apache/flink/runtime/scheduler/SchedulerBase.java

```diff
@@ -1020,7 +1020,7 @@ private void startAllOperatorCoordinators() {
 		catch (Throwable t) {
 			ExceptionUtils.rethrowIfFatalErrorOrOOM(t);
 			coordinators.forEach(IOUtils::closeQuietly);
-			throw new FlinkRuntimeException("Failed to start the operator coordinators", t);
+			failJob(new FlinkRuntimeException("Failed to start the operator coordinators", t));
```

Comment:
I am not sure why we have this change in this PR.

Reply:
oh damn. I was trying something out. I should not use `git commit -am` ;) 
Thanks a lot for this important catch!